tests/test_output.py

```diff
@@ -164,3 +166,42 @@ def test_sans_prefix(self):
             "%s/foo.txt" % output.key_prefix.lstrip('/'),
             extra_args={"ACL": "bucket-owner-full-control", "ServerSideEncryption": "AES256"},
         )
+
+
+class JSONFormatterTest(BaseTest):
+
+    log_inputs = [
```

Comment:
its pretty common to do something like this as a "table" test, but that means inputs and outputs in the same record, else folks have to have parse the test logic to understand how to extend, especially as these are mismatched on size.

https://github.com/golang/go/wiki/TableDrivenTests